cs/CMakeLists.txt

```diff
@@ -0,0 +1,39 @@
+# Included in ../CmakeLists.txt if csharp build is selected.
+
+find_package(SWIG 3.0 REQUIRED)
+
+message("Generating SWIG C# files...")
+
+# Cleaning
+execute_process(COMMAND powershell "Remove-Item ${CMAKE_SOURCE_DIR}/cs/src/* -recurse -exclude .gitignore")
+
+set (SWIG_DEFINITION "")
+if (WITH_RESQML2_2)
+	string(APPEND SWIG_DEFINITION "-DWITH_RESQML2_2")
+endif (WITH_RESQML2_2)
+if (WITH_ETP)
+	string(APPEND SWIG_DEFINITION " -DWITH_ETP")
```

Comment:
Is it really possible not to be WITH_ETP?

Reply:
Of course not. "-DWITH_RESQML2_2" is not used as well.